hasura.planx.uk/migrations/1743429247515_run_sql_migration/up.sql

```diff
@@ -0,0 +1,34 @@
+CREATE OR REPLACE VIEW "public"."teams_summary" AS 
+ SELECT t.id,
+    t.name,
+    t.slug,
+    ts.reference_code,
+    ts.homepage,
+    t.domain AS subdomain,
+    ti.has_planning_data AS planning_data_enabled,
+    ts.has_article4_schema AS article_4s_enabled,
+    jsonb_build_object('helpEmail', ts.help_email, 'helpPhone', ts.help_phone, 'emailReplyToId', ts.email_reply_to_id, 'helpOpeningHours', ts.help_opening_hours) AS govnotify_personalisation,
+        CASE
+            WHEN (COALESCE(ti.production_govpay_secret, ti.staging_govpay_secret) IS NOT NULL) THEN true
+            ELSE false
+        END AS govpay_enabled,
+    ts.submission_email AS send_to_email_address,
+    COALESCE(ti.production_bops_submission_url, ti.staging_bops_submission_url) AS bops_submission_url,
+    tt.logo,
+    tt.favicon,
+    tt.primary_colour,
+    tt.link_colour,
+    tt.action_colour,
+        CASE
+            WHEN ((COALESCE(ti.production_file_api_key, ti.staging_file_api_key) IS NOT NULL) AND (COALESCE(ti.production_power_automate_api_key, ti.staging_power_automate_api_key) IS NOT NULL) AND (ti.power_automate_webhook_url IS NOT NULL)) THEN true
+            ELSE false
+        END AS power_automate_enabled,
+    array_agg(f.name ORDER BY f.name) FILTER (WHERE f.status = 'online'::text AND f.deleted_at IS NULL) AS live_flows
```

Comment:
An additional check has been added to the filter here to exclude archived flows.